docker-compose/merging/docker-compose.override.yml

```diff
@@ -74,3 +74,10 @@ services:
       - $PWD/../env.json:/usr/local/apache2/htdocs/gridmerge/env.json:Z
     depends_on:
       - logspout
+
+  cgmes-boundary-import-job:
+    build: merging/cgmes-boundary-import-job
+    volumes:
+      - $PWD/cgmes-boundary-import-job/job-crontab:/root/job-crontab:Z
+      - $PWD/cgmes-boundary-import-job/cgmes-boundary-import-job-config.yml:/root/.itools/config.yml:Z
+
```

Comment:
The volumes creation does ot work in the profile 'suite' (all).
The `docker-compose/merging/cgmes-boundary-import-job` directory  must be moved to `docker-compose` directory
and modify the volumes declaration like this :
```
    volumes:
      - $PWD/../gridmerge-app-idpSettings.json:/usr/local/apache2/htdocs/gridmerge/idpSettings.json:Z
      - $PWD/../env.json:/usr/local/apache2/htdocs/gridmerge/env.json:Z
```

Reply:
Done